.env.example

```
@@ -26,36 +29,30 @@ PUBLIC_AUTH_REFRESH_TOKEN_TTL_SECONDS=604800
-RAZORPAY_KEY_ID=rzp_testx_xxxx
-RAZORPAY_KEY_SECRET=xxxx
-RAZORPAY_PLAN_ID=xxxx
-RAZORPAY_WEBHOOK_SECRET=xxxx
-
-# ── Internal Microservices (AI & Python) ──────────────────────────────────────
-# Required for AI Query Builder and Python microservice communication
-PYTHON_SERVICE_URL=http://localhost:8000
-INTERNAL_SECRET=generate_a_random_32_char_secret_here
-GROQ_API_KEY=gsk_your_groq_api_key_here
+# Get from: https://dashboard.razorpay.com → Settings → Webhooks → Add Endpoint
+RAZORPAY_WEBHOOK_SECRET=whsec_xxxxxxxxxxxx
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)